tensorboard/plugins/profile/overview_page/overview-page.html

```diff
@@ -390,21 +425,21 @@
                  'Other useful resources');
        }
      },
-
-     _onErrorChanged: function() {
-       this._updateView();
+     _showErrorMessage: function(msg) {
+         this._show_error_message = (msg != '');
```

Comment:
can `_show_error_message` be a computed property?

Reply:
Done.